tests/console/validate_filesystem_first_boot.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary:
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+sub run {
+
+    select_console "root-console";
+    my $test_data  = get_test_suite_data();
+    my @partitions = @{$test_data->{partitions}};
+    # Validate that the partition table type is the expected one.
+    script_run("lsblk");
+    script_run("fdisk -l");
+    script_run("parted -l");
```

Comment:
I extracted that part in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/dc249118b5fd547e533b93e54d0d763780404659/lib/validate_partitions.pm, hopefully that PR will be merged soon, so you can also put you common functions there, as we are starting to have a lot of duplicated code in those validate modules